Bottom: 29f7286704569501e768c26716f57dc0a244e23d
Top:    45950061e96f3725b666bbe4061f648f52898e90
Author: Guy Shapiro <guysh@mellanox.com>
Date:   2013-11-05 19:52:20 +0200

librdmacm: Some fixes to man pages

Fix the man pages of rdma_destroy_ep & rdma_destroy_qp to the correct return value (void).

Signed-off-by: Sean Hefty <sean.hefty@intel.com>


---

diff --git a/man/rdma_destroy_ep.3 b/man/rdma_destroy_ep.3
index b48a1e5..750702a 100644
--- a/man/rdma_destroy_ep.3
+++ b/man/rdma_destroy_ep.3
@@ -5,16 +5,13 @@ rdma_destroy_ep \- Release a communication identifier.
 .SH SYNOPSIS
 .B "#include <rdma/rdma_cma.h>"
 .P
-.B "int" rdma_destroy_ep
+.B "void" rdma_destroy_ep
 .BI "(struct rdma_cm_id *" id ");"
 .SH ARGUMENTS
 .IP "id" 12
 The communication identifier to destroy.
 .SH "DESCRIPTION"
 Destroys the specified rdma_cm_id and all associated resources
-.SH "RETURN VALUE"
-Returns 0 on success, or -1 on error.  If an error occurs, errno will be
-set to indicate the failure reason.
 .SH "NOTES"
 rdma_destroy_ep will automatically destroy any QP and SRQ associated with
 the rdma_cm_id.
diff --git a/man/rdma_destroy_qp.3 b/man/rdma_destroy_qp.3
index aeff667..bb1360e 100644
--- a/man/rdma_destroy_qp.3
+++ b/man/rdma_destroy_qp.3
@@ -11,9 +11,6 @@ rdma_destroy_qp \- Deallocate a QP.
 RDMA identifier.
 .SH "DESCRIPTION"
 Destroy a QP allocated on the rdma_cm_id.
-.SH "RETURN VALUE"
-Returns 0 on success, or -1 on error.  If an error occurs, errno will be
-set to indicate the failure reason.
 .SH "NOTES"
 Users must destroy any QP associated with an rdma_cm_id before
 destroying the ID.
